### PR TITLE
Moves inclusion of AdminMiddleware in Admin panel below layout definition

### PR DESCRIPTION
**Pull request by WendyBeth**
So that the layout can be customized.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.3.1
+2.3.0
```
Comment (JesseHerrick) [inline .ruby-version]: Why move backward?

**Comment by JesseHerrick** [comment]
Overall looks good @WendyBeth, but something is failing on the CI.

File: .ruby-version (hunk 1 above)
Comment (WendyBeth) [inline .ruby-version]: I guess I didn't need to commit this. rvenv doesn't recognize 2.3.1 yet and I wasn't willing to fight with it.

Comment (JesseHerrick) [inline .ruby-version]: It should be there if you upgrade rbenv (homebrew or git) because I've been using 2.3.1 since it came out.

**Pull request merged**